harambe/core.py

```diff
@@ -359,9 +359,18 @@ async def run(
             )
             if setup:
                 await setup(sdk)
-
             if not harness_options.get("disable_go_to_url", False):
                 await page.goto(url)
+                if harness == playwright_harness:
+                    await page.wait_for_load_state("domcontentloaded")
+                    await page.evaluate("""
+                            document.addEventListener('click', (event) => {
+                                const target = event.target.closest('a[href^="mailto:"], a[href^="tel:"]');
+                                if (target) {
+                                    event.preventDefault();
+                                }
+                            });
```

Comment:
why do we need this if we have the route handler already?

Reply:
because the route handler will prevent the request from being sent, but it will not prevent the js code from being executed so the popup window will still appear, using this code, it will prevent the js code itself from opening a new window since its href starts with `mailto` or `tel`. @asim-shrestha